### PR TITLE
Remove delegation, add candidate verification

### PR DESCRIPTION
**Pull request by bremoran**

### CODE REVIEW

File: draft-ietf-suit-trust-domains.md [hunk 1]
```diff
@@ -636,15 +646,6 @@ bz/m4rVlnIXbwK07HypLbAmBMcCjbazR14vTgdzfsJwFLbM5kdtzOLSolg==
...
-## Example 0: Delegation Chain
-
-This example uses functionalities:
-
-* manifest component id
-* delegation chain
-
-{::include examples/example0_delegation.txt}
-
 ## Example 1: Process Dependency
```
Comment (kentakayama) [inline draft-ietf-suit-trust-domains.md:649]: @bremoran 
Could you update the numbers?
Example 1 => 0
Example 2 => 1 (in L.659)

Comment (bremoran) [reply draft-ietf-suit-trust-domains.md:649]: Will do!

**Review by SylwesterKonczyk — APPROVED** [review]
Regarding suit-candidate-verification - that is exactly what is needed.

File: draft-ietf-suit-trust-domains.md (hunk 1 above)
Comment (kentakayama) [reply draft-ietf-suit-trust-domains.md]: Thanks!

**Pull request merged**